src/controllers/rings-by-controller.ts

```diff
@@ -21,10 +30,22 @@ export default class RingsByController extends AbstractController {
 
   @GET
   @Path('/')
-  @Response<Ring[]>(200, 'List of all available rings.')
+  @Response<RingListResponse>(200, 'List of all available rings.')
```

Comment:
Please add annotations for each endpoint

Reply:
what kind of annotations did you mean?